nisystemlink/clients/dataframe/models/_data_frame.py

```diff
@@ -52,6 +53,31 @@ class DataFrame(JsonModel):
     columns: Optional[List[str]] = None
     """The names and order of the columns included in the data frame."""
 
-    data: List[List[Optional[str]]]
+    data: Optional[List[List[Optional[str]]]] = None
```

Comment:
Could making this field optional lead to an error? This will provide a way to call the API without providing data.